front/app/api/analysis_tags/types.ts

```diff
@@ -0,0 +1,48 @@
+import { Keys } from 'utils/cl-react-query/types';
+import tagsKeys from './keys';
+
+export type TagsKeys = Keys<typeof tagsKeys>;
+
+export interface ITagParams {
+  analysisId: string;
+}
+
+export type TagType = 'custom';
```

Comment:
Still needs expanding, just making sure you didnt' look over it

Reply:
This type will be expanded with the magic tag types when we get to them.